### PR TITLE
Fix typo in netci.groovy

### PR DESCRIPTION
**Pull request by iamjasonp**

### CODE REVIEW

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by roncain** [comment]
LGTM subject to CI

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
@mmitche, could you also take a look please?

**Comment by iamjasonp** [comment]
Spot checked a windows, mac, and ubuntu queue, and things look ok

File: netci.groovy [hunk 1]
```diff
@@ -181,31 +181,28 @@ branchList.each { branchName ->
...
+                Utilities.setMachineAffinity(newJob, os, 'latest-or-auto')
+            }
 
             // Set up standard options.
             Utilities.standardJobSetup(newJob, project, isPR, getFullBranchName(branchName))
             // Add the unit test results
             Utilities.addXUnitDotNETResults(newJob, 'bin/tests/**/testResults.xml')
             
+            // Due to limitations with how the CI server works, we have to disable concurrent builds for outerloops 
+            newJob.concurrentBuild(false)
```
Comment (mmitche) [inline netci.groovy]: What's the reason for disabling the concurrency?

Comment (iamjasonp) [inline netci.groovy]: Our outerloops rely on us calling an HTTP endpoint to sync server code, after which the client will test against the assigned server. 

At the moment, we have a design limitation where one server endpoint is assigned to one CI queue; if we allow concurrent builds, it becomes possible to pave over the server endpoint with mismatched code while another test is running.

Comment (mmitche) [inline netci.groovy]: Sounds good, LGTM

Comment (iamjasonp) [inline netci.groovy]: I'll update the comment in the netci.groovy file and then merge. thanks!

Comment (mmitche) [inline netci.groovy]: It seems like we need a better solution for this eventually. I can see lots of places where we would want to deploy multiple servers (developer scenarios for instance) concurrently.

**Comment by iamjasonp** [comment]
Just a final check due to a comment change
@dotnet-bot test ci please

File: netci.groovy (hunk 1 above)
Comment (iamjasonp) [inline netci.groovy]: I'll open an issue on this for consideration

**Pull request merged**